src/main/webapp/app/view/project/admin/project-admin-update.component.html

```diff
@@ -51,6 +53,33 @@ <h2 id="jhi-project-heading">
                         </small>
                     </div>
                 </div>
+
+                <div *appHasAnyRole="{projectId:project!.id!,roles:roleAdmin}" class="form-group">
```

Comment:
```suggestion
                <div *appHasAnyRole="{ projectId:project!.id!, roles:roleAdmin }" class="form-group">
```
maybe look how other `appHasAnyRole` were formatted.